rss_pipeline/transform_rss.py

```diff
@@ -0,0 +1,188 @@
+"""
+This script converts the XML to dataframes and cleans them 
+as well as applying sentiment analysis
+"""
+
+from datetime import datetime
+import xml.etree.ElementTree as ET
+import pandas as pd
+import nltk
+from nltk.sentiment.vader import SentimentIntensityAnalyzer
+import requests
```

Comment:
`requests` is built-in and should appear between `datetime` and `xml.etree.ElementTree`